### PR TITLE
Add page up/down.

### PR DESCRIPTION
**Pull request by Qingtian-Zou**
Add page up/down.

Thanks to [Mpv-Playlistmanager](https://github.com/jonniek/mpv-playlistmanager) for inspiration.

### CODE REVIEW

File: navigator.lua [hunk 1]
```diff
@@ -191,6 +191,24 @@ function navup()
   handler()
 end
 
+function movepageup()
+  if cursor == 0 then return end
+  local prev_cursor = cursor
+  cursor = cursor - settings.visible_item_count
+  if cursor < 0 then cursor = 0 end
+  if selection then mp.commandv("playlist-move", prev_cursor, cursor) end
```
Comment (jonniek) [inline navigator.lua]: This line from playlistmanager is probably something that filenavigator shouldn't do 🤔 (same for pagedown)

File: navigator.lua [hunk 2]
```diff
@@ -24,16 +24,16 @@ local settings = {
...
   defaultpath = windows_desktop or os.getenv("HOME") or "/",
-  forcedefault = false, --force navigation to start from defaultpath instead of currently playing file
+  forcedefault = true, --force navigation to start from defaultpath instead of currently playing file
   --favorites in format { 'Path to directory, notice trailing /' }
   --on windows use double backslash c:\\my\\directory\\
   favorites = {
-    '/media/HDD2/music/music/',
-    '/media/HDD/users/anon/Downloads/',
-    '/home/anon/',
+    'y:\\Anime\\',
```
Comment (jonniek) [inline navigator.lua]: Could you undo the changes to settings as they are not relevant for the PR

File: README.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+Forked from [Mpv-Playlistmanager](https://github.com/jonniek/mpv-playlistmanager). Add page up/down.
+
+--------------------------------
+
```
Comment (jonniek) [inline README.md]: This attribution can be removed as both scripts are made by me and are using unlicense.

File: navigator.lua (hunk 1 above)
Comment (Qingtian-Zou) [reply navigator.lua]: line removed

File: navigator.lua (hunk 2 above)
Comment (Qingtian-Zou) [reply navigator.lua]: Reverted back to original default configurations.

File: README.md (hunk 3 above)
Comment (Qingtian-Zou) [reply README.md]: Changes to README.md are dropped.

**Comment by jonniek** [comment]
Thanks! Sorry for missing this PR, something wrong with GH notifications.
